### DISCUSSION TITLE
current transaction is aborted, commands ignored until end of transaction block

### DISCUSSION THREAD

**Discussion by danilsizov** [supabase/supabase#11437] [Questions]
Today, according to the logic I do not understand, I began to receive such an error. Moreover, I don’t get it all the time, I can’t catch it. I didn’t change anything in the database, and I didn’t touch the settings, but the error started to appear, and if I send the same request a couple more times, I normally get a response without an error.

Log ID
b5d5ae8c-d682-418a-9257-582b1d5886c3

Log Timestamp (UTC)
2023-01-04T02:44:36.695Z

Log Event Message
current transaction is aborted, commands ignored until end of transaction block

Log Metadata
[
  {
    "file": "/var/log/postgresql/postgresql.csv",
    "parsed": [
      {
        "application_name": "postgrest",
        "backend_type": "client backend",
        "command_tag": "BIND",
        "connection_from": "127.0.0.1:46850",
        "context": null,
        "database_name": "postgres",
        "detail": null,
        "error_severity": "ERROR",
        "hint": null,
        "internal_query": null,
        "internal_query_pos": null,
        "leader_pid": null,
        "location": null,
        "process_id": 149644,
        "query": "BEGIN ISOLATION LEVEL READ COMMITTED READ WRITE",
        "query_id": 0,
        "query_pos": null,
        "session_id": "63b4c94f.2488c",
        "session_line_num": 48,
        "session_start_time": "2023-01-04 00:33:19 UTC",
        "sql_state_code": "25P02",
        "timestamp": "2023-01-04 02:44:36.695 UTC",
        "transaction_id": 0,
        "user_name": "authenticator",
        "virtual_transaction_id": "21/0"
      }
    ],
    "parsed_from": null,
    "source_type": "file"
  }
]

**Comment by 0xwurzel** [comment]
FWIW (not much probably), I've been having the same problem for the last 24 hours. I've just restarted the project (in Settings | General) and it seems to have fixed the issue. Not ideal but I thought I'd let you know
